src/main/services/window-manager.ts

```diff
@@ -119,6 +125,7 @@ export class WindowManager {
     this.isReadyToShowMainWindow = true;
     this.splashWindow?.close();
     this.createMainWindow();
+    if (this.isMainMaximize) this.mainWindow?.maximize();
```

Comment:
Move this conditional inside the createMainWindow function (after `this.mainWindow.removeMenu();` might be fine)

Reply:
Good idea